climakitae/meteo_yr.py

```diff
@@ -1,12 +1,7 @@
 """
-Calculates the Average Meterological Year (TMY) for the Cal-Adapt: Analytics
-Engine using a standard climatological period (1981-2010) for the historical
-baseline, and uses a 30-year window around when a designated warming level is
-exceeded for the SSP3-7.0 future scenario for 1.5°C, 2°C, and 3°C. Additional
-functionality for 4°C is forthcoming. Working Group 4 (Aug 31, 2022) version
-focuses on air temperature and relative humidity, with all variables being
-available for Analytics Engine Beta Launch. The AMY is comparable to a typical
- meteorological year, but not quite the same full methodology.
+Calculates the Average Meterological Year (AMY) and Severe Meteorological Year (SMY) for the Cal-Adapt: Analytics Engine using a standard climatological period (1981-2010) for the historical baseline, and uses a 30-year window around when a designated warming level is exceeded for the SSP3-7.0 future scenario for 1.5°C, 2°C, and 3°C. 
+
+Additional functionality for 4°C is forthcoming. Working Group 4 (Aug 31, 2022) version focuses on air temperature and relative humidity, with all variables being available for Analytics Engine Beta Launch. The AMY is comparable to a typical meteorological year, but not quite the same full methodology.
```

Comment:
IMO we should/can remove the "additional functionality for 4°C...... WG4 all variables being available for AE Beta launch" sentences. First sentence has been backburnered, and second is out-of-date now. I'd keep the third sentence about AMY being somewhat comparable to TMY, but that's a conversation for later too.

Reply:
Sounds good, thanks for taking a look at the description. I figured some of it was outdated but wasn't sure what part.